$NetBSD: patch-ai,v 1.1.1.1 2010/10/19 16:37:40 abs Exp $

# Use CXX rather than CC to link c++, to ensure correct handling on more platforms

--- src/modules/qimage/Makefile.orig	2010-09-14 06:00:09.000000000 +0000
+++ src/modules/qimage/Makefile
@@ -24,7 +24,7 @@ SRCS := $(OBJS:.o=.c) $(CPPOBJS:.o=.cpp)
 all: 	$(TARGET)
 
 $(TARGET): $(OBJS) $(CPPOBJS)
-		$(CC) $(SHFLAGS) -o $@ $(OBJS) $(CPPOBJS) $(LDFLAGS)
+		$(CXX) $(SHFLAGS) -o $@ $(OBJS) $(CPPOBJS) $(LDFLAGS)
 
 depend:	$(SRCS)
 		$(CC) -MM $(CFLAGS) $(QTCXXFLAGS) $^ 1>.depend
